src/Android/Avalonia.Android/Platform/SkiaPlatform/AndroidFramebuffer.cs

```diff
@@ -32,7 +32,7 @@ public AndroidFramebuffer(Surface surface, double scaling)
             RowBytes = buffer.stride * (Format == PixelFormat.Rgb565 ? 2 : 4);
             Address = buffer.bits;
 
-            Dpi = scaling * new Vector(96, 96);
```

Comment:
Why is this change required?

Reply:
Because i do receive compiler error. It looks like `scaling * new Vector(96, 96)` returns `double` not `Vector` & `new Vector(96, 96) * scaling`.